samplecode/example/pom.xml

```diff
@@ -3,8 +3,8 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 
-	<artifactId>spring-data-documentdb-example</artifactId>
-	<name>Spring Data DocumentDb - Example</name>
+	<artifactId>spring-data-cosmosdb-example</artifactId>
+	<name>Spring Data Cosmosdb - Example</name>
 
 	<parent>
 		<groupId>pli.spring.data.documentdb.examples</groupId>
```

Comment:
Also the sample code, Sorry I forgot the remove my name as prefix here.
`pli.spring.data.documentdb.examples` -> `spring.data.cosmosdb.examples`. Thanks a lot.

Reply:
Fixed the sample pom file with [new commit](https://github.com/Microsoft/spring-data-documentdb/pull/85/commits/b82b0a25fe5d1f86f24cd171cf4c64d0bc468e30)